webapp/views/registration.pug

```diff
@@ -0,0 +1,5 @@
+html
```

Comment:
This template should extend from the base template, similar to how /views/index.pug does. 

It's also not rendering anything because the route controller isn't sending the right variables, but that's not an issue.